app/controllers/docs.php

```diff
@@ -27,6 +27,7 @@ private function edit() {
             $name = htmlspecialchars($_POST["name"]);
             $slug = htmlspecialchars(@$_POST["slug"] ?: "");
             $data = htmlspecialchars(@$_POST["data"] ?: "");
+            $comment = htmlspecialchars($_POST["comment"]);
```

Comment:
This might give an error. The same code is run when updating directory information, where we don't give comment right now.